Automattic/wp-calypso
# Implement migration instructions layout without site preview

Related to #92499

## Proposed Changes

* Implement a new design for the case without site preview.

## Why are these changes being made?


* Users are able to not inform the URL of the site they are migrating from, so we don't have a preview to display. In this case, we need to display the screen in a different way.

## Testing Instructions



* Make sure you have the `migration-flow/new-migration-instructions-step` feature flag enabled.
* Create a new site with the Import intent.
* Navigate until the migration instructions step in 2 different ways:
  * Informing the site you are migrating from, or just informing that it's a WordPress site through the link: _"pick your current platform from a list"_.
* For both situations, test with different screen sizes and make sure the migration instructions screen matches the expected design. QnzGpMHVpKuf0ZeRQVcxiL-fi-7112_4639

## Screenshots

### With Preview

<img width="498" alt="Screenshot 2024-07-15 at 19 55 45" src="https://github.com/user-attachments/assets/002c2302-ad64-4020-a8d7-a728778b25f7">

<img width="1271" alt="Screenshot 2024-07-15 at 19 55 31" src="https://github.com/user-attachments/assets/24794660-ed46-4e42-ab66-ac57ac45ec7a">

### Without preview

<img width="500" alt="Screenshot 2024-07-15 at 19 55 54" src="https://github.com/user-attachments/assets/741c1029-0cef-4324-971f-8980d6c28935">

<img width="1271" alt="Screenshot 2024-07-15 at 19 55 22" src="https://github.com/user-attachments/assets/60df8c1c-e221-4c02-888f-52eff8ec4223">

## Pre-merge Checklist
